libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -151,8 +151,8 @@ func (m *Manager) Apply(pid int) error {
 		properties []systemdDbus.Property
 	)
 
-	if c.Slice != "" {
-		slice = c.Slice
+	if c.Parent != "" {
```

Comment:
But Docker can not be the only user, if we enable systemd cgroup for runc, runc itself can set `c.Parent` to anything.